.packit.yaml

```diff
@@ -1,18 +1,37 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: dnf5.spec
+
+# add or remove files that should be synced
 files_to_sync:
   - dnf5.spec
+  - .packit.yaml
+
+actions:
+  fix-spec-file:
+    # set the correct version for global project_version_{major,minor,patch} macros needed by %build
+    bash -c "sed -i -r \"s/^%global project_version_major (\s*)\S+/%global project_version_major \1`echo ${PACKIT_PROJECT_VERSION} | cut -d. -f3`/\" dnf5.spec"
```

Comment:
I think the requested field index is incorrect here, should be 1:  "... cut -d. **-f1** ..."